.github/workflows/nightly.yaml

```diff
@@ -172,7 +172,7 @@ jobs:
         with:
           fetch-depth: 0
       - name: Kubernetes ${{ matrix.kubernetes_version }} ${{ matrix.dbmode }} Integration Tests
-        run: KONG_CLUSTER_VERSION=${{ matrix.kubernetes_version }} make test.integration.${{ matrix.dbmode }}
+        run: TEST_KONG_CONTROLLER_IMAGE_OVERRIDE="kong/nightly-ingress-controller:nightly-redhat" KONG_CLUSTER_VERSION=${{ matrix.kubernetes_version }} make test.integration.${{ matrix.dbmode }}
```

Comment:
Don't recall any particular reason I used that one instead of distroless. distroless can be harder to debug because you can't shell into it and install diagnostic tools, but that doesn't really matter for CI.

Reply:
```suggestion
        run: TEST_KONG_CONTROLLER_IMAGE_OVERRIDE="kong/nightly-ingress-controller:nightly" KONG_CLUSTER_VERSION=${{ matrix.kubernetes_version }} make test.integration.${{ matrix.dbmode }}
```

This change is correct for actual E2E but does nothing here. Separating #2788 from #2790 was perhaps ill-advised because the E2E tests have considerably different requirements: integration never uses any controller image because it runs controller code directly from the test suite, and this variable does nothing when running the integration tests. It does work as a sort of placeholder, I suppose.

Reply:
Thanks for your suggestion. LMK how I can make it work. I mean where I should declare that var.